### ISSUE TITLE
[feature request] chronological (sub)playlist sort

### ISSUE THREAD

**Issue by RJVB** [zaps166/QMPlay2#552]
It would be useful if one could sort items in a playlist (or in individual playlist folders/groups) chronologically. Ideally this would also sort YT videos by upload/creation date.

BTW; do the `sort` options apply to the selected folders/groups or to the entire current playlist? Mine is too large to try and risk messing it up.

**Comment by zaps166** [comment]
> do the sort options apply to the selected folders/groups or to the entire current playlist?

If you select a single group, it should apply to this group only. If you do it outside the group, it'll sort everything.

> Mine is too large to try and risk messing it up.

Backup the `~/.config/QMPlay2/Playlist.pls` and try :smile: Or do it and kill the process - in this case QMPlay2 will not save the playlist.

**Comment by RJVB** [comment]
> Or do it and kill the process - in this case QMPlay2 will not save the playlist.

Yeah, I found that out yesterday, in a context where I would have preferred saving...

**Comment by RJVB** [comment]
FWIW, youtube videos have the upload date listed as `upload_date` in the json data obtained via yt-dl (format yyyymmdd). This data is included in the json dump obtained for playlist URLs.
